### ISSUE TITLE
Enable `allowfullscreen` for all `amp-iframe` cases

### ISSUE THREAD

**Issue by aghassemi** [ampproject/amphtml#20306]
I have been running into multiple cases where an `amp-iframe` is used to embed a video player but because the developer/CMS has neglected adding `allowfullscreen` attribute, the fullscreen button for video is disabled causing bad UX.

I like to propose we auto-enabled `allowfullscreen` for all `amp-iframe` cases. This needs to be reviewed by the larger group but given fullscreen API can only be triggered via user action, I can not see the risks of doing this.

Note that we already do this for `amp-video-iframe` which semantically is the same as an `amp-iframe`.

**Comment by ampprojectbot** [comment]
This issue doesn't have a category which makes it harder for us to keep track of it. @aghassemi Please add an appropriate category.

**Comment by ampprojectbot** [comment]
This issue doesn't have a category which makes it harder for us to keep track of it. @aghassemi Please add an appropriate category.

**Comment by aghassemi** [comment]
@ampproject/tsc any concerns with this proposal?

**Comment by cramforce** [comment]
My initial reaction is that we probably should not do this.

Changing defaults with possible security impact (untrusted frame may fake URL bar).

**Comment by aghassemi** [comment]
Maybe a different approach is better here: Run some basic heuristics to detect if iframe is pointing to video and do developer warning to switch to `amp-video-iframe`...

**Comment by aghassemi** [comment]
Closing. Not gonna happen.
